src/tendisplus/commands/command.cpp

```diff
@@ -874,17 +874,14 @@ std::vector<int> Command::getKeysFromCommand(
 }
 
 bool Command::isAdminCmd(const std::string& cmd) {
-  static const auto sAdmin = []() {
```

Comment:
@KunWang2 we use static variable to get lambda call result here. It means that after the first time `isAdminCmd` called. this `tmp` will be static variable.It'will only call `std::unordered_set::count` later.It's complexity about `O(1)`. But `std::map::find` 's complexity is about `O(logN)`